docs/bnb_greenfield/guide/getting-started/ecosystem.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Ecosystem
+description: An explaination of BNB Greenfield ecosystem, including the contributions and responsibilities of each group involved.  
+keywords: [BNB Greenfield ecosystem, validators, storage providers, dapps, challengers, verifiers]
+order: 2
+---
+
+# Ecosystem
```

Comment:
I think `ecosystem` is not belong to `getting-started`, it is a very general introduction

Reply:
Done by removing the page /getting-started/overview.md